pkg/config/config_windows.go

```diff
@@ -28,6 +30,7 @@ func osinit() {
 		defaultConfdPath = filepath.Join(pd, "conf.d")
 		defaultAdditionalChecksPath = filepath.Join(pd, "checks.d")
 		defaultRunPath = filepath.Join(pd, "run")
+		defaultSecurityAgentLogFile = filepath.Join(pd, "security-agent.log")
```

Comment:
Shouldn't it be "logs", "security-agent.log" (like in `pkg/process/config/config_windows.go`) ?

Reply:
yes, your are right! thanks